src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -492,6 +492,27 @@ lemma inf_dist_image (hΦ : isometry Φ) :
   inf_dist (Φ x) (Φ '' t) = inf_dist x t :=
 by simp [inf_dist, inf_edist_image hΦ]
 
+/-! `inf_dist` as a function into `ℝ≥0`. -/
+
+/-- The minimal distance of a point to a set as a `nnreal` -/
+def inf_nndist (x : α) (s : set α) : ℝ≥0 := ennreal.to_nnreal (inf_edist x s)
+@[simp] lemma coe_inf_nndist : (inf_nndist x s : ℝ) = inf_dist x s := rfl
+
+-- @[simp] lemma inf_nndist_eq_zero : (inf_nndist x s : ℝ) = inf_dist x s := rfl
```

Comment:
did you mean to comment this out?

Reply:
Oops, I thought I wanted a lemma with that name, but didn't end up using it. Forgot to delete it. Thanks for catching!